homeassistant/components/wallbox/services.yaml

```diff
@@ -0,0 +1,39 @@
+get_schedules:
+  fields:
+    device_id:
+      required: true
+      selector:
+        device:
+          integration: wallbox
```

Comment:
Could this maybe become a calendar entity?

Reply:
I like the idea, but as you said on the comments below, let's split it on another PR. :-)